src/additional-components/Controls/index.tsx

```diff
@@ -73,8 +81,28 @@ const Controls: FC<ControlProps> = ({
 
   useEffect(() => {
     setIsVisible(true);
+    setInteractive?.(true);
   }, []);
 
+  // https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values
+  const isControlPressed = useKeyPress("Control")
+  const isCommandPressed = useKeyPress("Meta")
+  const isValidControlOrCommandPressPerOS = (OS === OSName.Windows && isControlPressed) || (OS === OSName.Mac && isCommandPressed)
+  const isPlusPressed = useKeyPress("=")
```

Comment:
typo

Reply:
= vs +

Reply:
the key for + is `=`! that surprised me too - I'll leave a comment when moving over the changes

Reply:
Ah ok that kind of makes sense. I wonder do they share the same key on every keyboard layout though? Seems odd that shift keys would be base key + shift as I know some of them at least do vary.

Reply:
Also what about numpad +

Reply:
Oh I see. So actually, we need both. '+' would handle numpad + as well as shift =, while '=' handles = without shift. The latter case is still kind of weird because some keyboards don't have + mapped to the = key (ie. the German qwertz layout) but that seems like an edge case we can live with.